tests/introspection/ApiProtocolTest.cs

```diff
@@ -122,6 +122,7 @@ protected virtual bool Skip (Type type, string protocolName)
 				case "NSMeasurement`1":
 				// iOS 11 / tvOS 11
 				case "VSSubscription":
+				case "HMAccessControl": // not in headers
```

Comment:
Minor but I guess you meant `Conformance not in headers`, this makes it sound like `HMAccessControl` is not available in the headers.

Reply:
should not be needed, it was added in the first PR (in iOSProtocolTest.cs)